src/constants.rs

```diff
@@ -9,6 +9,12 @@ pub const APP_INFO: &'static AppInfo = &AppInfo {
 
 /// The default API URL
 pub const DEFAULT_URL: &'static str = "https://sentry.io/";
+/// The default device family
+#[cfg(windows)]
+pub const DEFAULT_FAMILY: &'static str = "Windows device";
+#[cfg(not(windows))]
+pub const DEFAULT_FAMILY: &'static str = "Unix device";
```

Comment:
These default values do not really make sense as this information is already present in the `"os"` field. Just remove the defaults completely please.